UnityProject/Assets/Scripts/ScriptableObjects/Atmospherics/GasMixesSO.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Linq;
+using Systems.Atmospherics;
+using UnityEngine;
+
+namespace ScriptableObjects.Atmospherics
+{
+	[CreateAssetMenu(fileName = "GasMixesSO", menuName = "ScriptableObjects/Atmos/GasMixesSO")]
+	public class GasMixesSO : ScriptableObject
+	{
+		[SerializeField]
+		private GasMix gasMix = new GasMix();
+		public GasMix GasMix => gasMix;
```

Comment:
Might need some stuff so it doesn't just use the gas mix on the SO accidentally, Maybe like
copymix